crates/sui/src/client_ptb/ptb.rs

```diff
@@ -160,17 +160,19 @@ impl PTB {
         )
         .await?;
 
-        if let SuiClientCommandResult::DryRun(response) = transaction_response {
-            println!("{}", Pretty(&response));
-            return Ok(());
-        }
-
-        let transaction_response =
-            if let SuiClientCommandResult::TransactionBlock(response) = transaction_response {
-                response
-            } else {
-                anyhow::bail!("Internal error. Cannot run the PTB")
-            };
+        let transaction_response = match transaction_response {
+            SuiClientCommandResult::DryRun(response) => {
+                println!("{}", Pretty(&response));
```

Comment:
Could this part be handled by adding the logic to pretty-print `DryRun(response)` into the `Display` impl for `SuiClientCommandResult`?

Reply:
It's already there so easy fix.